### ISSUE TITLE
What is difference between tensorflow version and pytorch version?

### ISSUE THREAD

**Issue by Yangseung** [tkipf/pygcn#45]
What is difference between tensorflow version and pytorch version?
and why did you change? (split of training dataset and add dropout layer)

**Comment by bytes-by-talari** [comment]
To Optmize probably , I'm not sure but i think that is the reason.
